presto-tests/src/main/java/com/facebook/presto/tests/statistics/Metric.java

```diff
@@ -15,22 +15,42 @@
 
 import com.facebook.presto.cost.PlanNodeStatsEstimate;
 
-import java.util.function.Function;
+import java.util.Objects;
+import java.util.Optional;
 
-public enum Metric
+public abstract class Metric<T>
```

Comment:
No need for `T`. Currently all metrics return `double`.

Reply:
I removed the `T` from all related classes.